module/data/shared/activation-field.mjs

```diff
@@ -17,4 +17,29 @@ export default class ActivationField extends SchemaField {
     };
     super(fields, options);
   }
+
+  /* -------------------------------------------- */
+  /*  Data Preparation                            */
+  /* -------------------------------------------- */
+
+  /**
+   * Prepare data for this field. Should be called during the `prepareFinalData` stage.
+   * @this {ItemDataModel|BaseActivityData}
+   * @param {object} rollData  Roll data used for formula replacements.
+   * @param {object} [labels]  Object in which to insert generated labels.
+   */
+  static prepareData(rollData, labels) {
+    Object.defineProperty(this.activation, "scalar", {
```

Comment:
What's the benefit of a getter here, vs. derived data? It doesn't look like there's anything dynamic happening in the getter.

Reply:
Nothing then having it not being enumerable, but that probably isn't a huge benefit.

Reply:
Let's switch to derived values for now then, as a more understandable concept, unless we find a good reason to use getters.